### PR TITLE
Add package Garfield++ (garfieldpp)

### PR DESCRIPTION
**Pull request by mirguest**
Garfield++ is used to simulate the ionisation measurement in gases and semiconductors. 

Tao Lin

### CODE REVIEW

File: var/spack/repos/builtin/packages/garfieldpp/package.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    version('master', branch='master')
+    version('4.0', sha256='82bc1f0395213bd30a7cd854426e6757d0b4155e99ffd4405355c9648fa5ada3')
+    version('3.0', sha256='c1282427a784658bc38b71c8e8cfc8c9f5202b185f0854d85f7c9c5a747c5406')
+
+    depends_on('root')
+
+    def cmake_args(self):
+        args = []
+        return args
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/garfieldpp/package.py]: You can remove this method.

**Review by alalazo — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/garfieldpp/package.py (hunk 1 above)
Comment (mirguest) [reply var/spack/repos/builtin/packages/garfieldpp/package.py]: Done. Thanks.

**Review by alalazo — DISMISSED** [review]

File: var/spack/repos/builtin/packages/garfieldpp/package.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+class Garfieldpp(CMakePackage):
+    """Garfield++ is a toolkit for the detailed simulation of particle
+       detectors based on ionisation measurement in gases and semiconductors. """
+
+    homepage = "https://garfieldpp.web.cern.ch/garfieldpp/"
+    url      = "https://gitlab.cern.ch/garfield/garfieldpp/-/archive/4.0/garfieldpp-4.0.tar.gz"
+    git      = "https://gitlab.cern.ch/garfield/garfieldpp.git"
+
+    maintainers = ['mirguest']
+
```
Comment (vvolkl) [inline var/spack/repos/builtin/packages/garfieldpp/package.py:20]:
```suggestion

    tags = ['hep']

    maintainers = ['mirguest']

```

**Comment by mirguest** [comment]
Hi @alalazo ,
Could you help me merge this PR? Thank you very much!
Tao Lin

**Review by alalazo — APPROVED** [review]

**Pull request merged**